### PR TITLE
Add aspects

### LINKED ISSUE #24: Add aspects to core-css
The most common aspect sizes are:
9:16, 3:4, 1:1, 3:2, 4:3, 16:9
Suggestions for additional or removing aspects @mfolkeseth and @Isdoda ?

### PR DESCRIPTION
**Pull request by eirikbacker**
- Add aspects
- Fix typos
Resolves #24

### CODE REVIEW

File: dist/core-css.css [hunk 1]
```diff
@@ -167,6 +167,16 @@ button, select { cursor: pointer }
 }
 
 
+/* Aspects
+---------------------------------------------------------------- */
+.nrk-aspect { position: relative }
+.nrk-aspect > * { box-sizing: border-box; position: absolute; margin: 0; width: 100%; height: 100% }
+.nrk-aspect::after { content: ''; display: block; padding-top: 56.25% }
+.nrk-aspect--9x16::after { padding-top: 177.78% }
```
Comment (leftieFriele) [inline dist/core-css.css]: the -- usage is not consistent with how the nrk-xs-.. type selectors, wouldn't  .nrk-aspect-9x16 be more consistent with the rest of the module?

Comment (mfolkeseth) [reply dist/core-css.css]: +1

Comment (eirikbacker) [reply dist/core-css.css]: @leftieFriele Was thinking of this as an option :) but this implies photo sizes are changing on screen size change. That requires more work on a `picture` tag than just changing class, so implementing
```
@media (min-width:700px) {
  .nrk-aspect-4x3 { ... }
}
```
is possible for the specific use case. the `.nrk-xs-` vs `.nrk-aspect--` is very intentional: `.nrk-xs-` can be used without being put in a grid, while `.nrk-aspect--` will not have any effect without being used in conjunction with `.nrk-aspect` - following BEM guidelines :)

Comment (leftieFriele) [reply dist/core-css.css]: if we follow BEM guidelines then that's cool :+1:

**Pull request merged**